workshop-registration.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <title>WebDev Workshop Registration</title>
+    </head>
+    <body>
+        <h1 id="main-title">WebDev Workshop Registration</h1>
+        <p id="desc1">
+        Join our immersive workshop to sharpen your modern web development skills.<br>
```

Comment:
BR is not suitable here. If you want this text to show in 2 paragraphs, it should be in 2 separate P tags then.